components/TextFields/src/MDCTextInputControllerBase.h

```diff
@@ -104,4 +104,15 @@ extern const CGFloat MDCTextInputControllerBaseDefaultBorderRadius;
  */
 @property(nonatomic, assign) NSUInteger minimumLines;
 
+/**
```

Comment:
I think this should be universal. Put it in `MDCTextInputController`.

Reply:
We spoke about this offline before I pushed up the PR. You agreed that we should only make the change for Outlined and Filled styles at this point.  The reasoning, as I understood it, is that [Material Guidelines only recommend the Outlined and Filled styles](https://material.io/design/components/text-fields.html#usage) based on internal usability testing.

Has something changed so that we should be actively supporting the other styles with new feature development?

Reply:
After discussing, I've filed #4453 as a follow-up to migrate this property up to the root protocol. Due to the amount of testing that should be done across implementing classes, we decided not to add it to this PR.